ui/src/App.js

```diff
@@ -2,46 +2,70 @@ import "./App.css";
 import "../node_modules/bootstrap/dist/css/bootstrap.min.css";
 import Header from "./Components/Header/Header";
 import Sidebar from "./Components/Sidebar/Sidebar";
-import { Route, Switch } from "react-router-dom";
+import { Route, Switch, useHistory } from "react-router-dom";
 import Home from "./Components/Home/Home";
 import Report from "./Components/Report/Report";
 import Company from "./Components/Company/Company";
 import Employees from "./Components/Employees/Employees";
 import Settings from "./Components/Settings/Settings";
 import Documents from "./Components/Documents/Documents";
-import Appointment from "./Components/Appointment/appointment";
-import React, { useState } from "react";
+import { useEffect, useState } from "react";
 import Register from "./Components/LoginPage/Register";
 import Login from "./Components/LoginPage/Login";
 import ForgotPassword from "./Components/LoginPage/ForgotPassword";
+import MainPage from "./Components/MainPage/MainPage";
+import Mainbody from "./Components/MainPage/mainbody";
+import { useSelector } from "react-redux";
+import Appointment from "./Components/Appointment/appointment";
+import React from "react";
 import MyProfile from "./Components/My-Profile/My-Profile";
 import OfferLetter from "./Components/Documents/Appointment";
 import Notifications from "./Components/Notifications/Notifications";
 
 
+
 function App() {
-  const [sidebarstatus, setSidebarStatus] = useState(false);
-  const sidebarclass = sidebarstatus ? "sidebar" : "sidebar sidebar-collapse";
-  function handlesidebarstatus() {
-    setSidebarStatus((previousstate) => {
-      return !previousstate;
-    });
+  let userrouter=useHistory()
+const[sidebarstatus,setSidebarStatus] = useState(false);
+const islogdin= useSelector(action=>action.login.islogdin)
+const sidebarclass = sidebarstatus ? 'sidebar': "sidebar sidebar-collapse";
+function handlesidebarstatus(){
+  setSidebarStatus((previousstate)=>{
+    return !previousstate;
+  })
+}
+useEffect(()=>{
+  console.log(islogdin)
+  if(islogdin){
```

Comment:
Change the variable name to 'isUserLoggedIn'

Reply:
done